daily-patch-information/src/main/java/org/wso2/patchinformation/MainEmailSender.java

```diff
@@ -0,0 +1,132 @@
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+package org.wso2.patchinformation;
+
+import org.apache.log4j.Logger;
+import org.wso2.patchinformation.email.EmailContentCreator;
+import org.wso2.patchinformation.email.EmailSender;
+import org.wso2.patchinformation.exceptions.PatchInformationException;
+import org.wso2.patchinformation.jira.JIRAAccessor;
+import org.wso2.patchinformation.jira.JIRAIssue;
+import org.wso2.patchinformation.pmt.PmtAccessor;
+import org.wso2.patchinformation.properties.PropertyValues;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import static org.wso2.patchinformation.constants.EmailConstants.EMAIL_SUBJECT_CUSTOMER;
+import static org.wso2.patchinformation.constants.EmailConstants.EMAIL_SUBJECT_INTERNAL;
+import static org.wso2.patchinformation.constants.EmailConstants.MAIN_HEADER_CUSTOMER;
+import static org.wso2.patchinformation.constants.EmailConstants.MAIN_HEADER_END;
+import static org.wso2.patchinformation.constants.EmailConstants.MAIN_HEADER_INTERNAL;
+/**
+ * Sends 2 emails on behalf of the engineering efficiency team. One on Customer related JIRA issues,
+ * and the other on Proactive JIRA issues and their corresponding patch information.
+ */
+public class MainEmailSender {
+
+    private static final Logger LOGGER = Logger.getLogger(MainEmailSender.class);
+
+    public static void main(String[] args) {
+        try {
+            LOGGER.info("Executing process to send email on Internal JIRA issues.");
+            executeEmailSendingProcess(false);
+            LOGGER.info("Execution completed successfully.\n");
+        } catch (PatchInformationException e) {
+            LOGGER.error("Execution failed, process was not completed\n", e);
+        }
+        try {
+            LOGGER.info("Executing process to send email on Customer related JIRA issues.");
+            executeEmailSendingProcess(true);
+            LOGGER.info("Execution completed successfully.\n");
+        } catch (PatchInformationException e) {
+            LOGGER.error("Execution failed, process was not completed\n", e);
+        }
+    }
+
+    /**
+     * The process gets internal JIRA issues or customer related JIRA issues, and sends an email
+     * containing information on them and their associated patches.
+     *
+     * @param isMailOnCustomerReportedIssues boolean to determine if it is the internal or customer related mail
+     *                                       being sent.
+     * @throws PatchInformationException The process execution has halted.
+     */
+    private static void executeEmailSendingProcess(boolean isMailOnCustomerReportedIssues)
```

Comment:
This is a long method, just split accessing JIRA and accessing PMT methods to two private methods, and call.